.rubocop.yml

```diff
@@ -1,522 +1 @@
----
-require:
-- rubocop-performance
-- rubocop-rspec
-AllCops:
-  DisplayCopNames: true
-  TargetRubyVersion: '2.6'
-  SuggestExtensions: false
-  Include:
```

Comment:
I notice these 2 lines aren't included in the new `rubocop_baseline.yml`. Is this because Rubocop will inspect all ruby files by default?

Reply:
whoops

Reply:
Oh hold on, I was confused by the diff.

I've removed the "include" property.